tools/yaml-templates/ios-test.yml

```diff
@@ -82,11 +82,9 @@ steps:
       -parallel-testing-worker-count 2
       -destination 'platform=iOS Simulator,name=iPhone 14 Pro,OS=16.2'
       -screenshot-enabled="YES"
-      -test-iterations 3
```

Comment:
just notice and wondering, are you removing retry?

Reply:
Remove it so that I could gather more info to figure out what happened in the background.